docs/components.d.ts

```diff
@@ -6,6 +6,8 @@ declare module 'vue' {
   export interface GlobalComponents {
     Avatar: typeof import('./src/components/Avatar.vue')['default']
     Contributors: typeof import('./src/components/Contributors.vue')['default']
+    CustomNavBar: typeof import('./src/components/CustomNavBar.vue')['default']
+    CustomNavBarTitle: typeof import('./src/components/CustomNavBarTitle.vue')['default']
```

Comment:
What is this? 👀

Reply:
upps, sorry, something I was trying... removing them

Reply:
don't remove spaces on `config.ts` for releases and contributing link, I'll make a PR upstream to use `&#160;` between text and external icon instead using space on the template.